examples/demos/devices_users_demo/main.go

```diff
@@ -42,12 +45,12 @@ func createDevice(client *kentikapi.Client) models.ID {
 		models.ID(11466),
 		models.CDNAttributeYes,
 	)
-	models.SetOptional(&device.SiteID, 8483)
-	models.SetOptional(&device.DeviceBGPFlowSpec, true)
+	device.SiteID = models.IDPtr(8483)
+	device.DeviceBGPFlowSpec = pointer.ToBool(true)
```

Comment:
Cannot the value be directly assigned? Most reasonable programming languages represent bool as some atomic type.

Reply:
We use pointers to achieve optional values. `DeviceBGPFlowSpec` field has `*bool` type. Pointer to bool must be assigned here. Such literals as "true" are not addressable, so it cannot be assigned inline. In other words, you cant do `device.DeviceBGPFlowSpec = &true`.

See: https://stackoverflow.com/a/30716481/5000820